### PR TITLE
Upgrade to Scala 2.13.12

### PR DESCRIPTION
**Pull request by ioannakok**
This commit also fixes "Implicit definition should have explicit type" errors. These are warnings to ease migration to Scala 3 but we have `-Xfatal-warnings` enabled in our `scalacOptions` in `ProjectSettings.scala`.

## Checklist

- [ ] Tested locally, and on CODE if necessary
- [ ] Will not break dotcom-rendering
- [ ] Will not break our database – if updating CAPI, [updated and committed the database files](https://github.com/guardian/frontend/blob/main/docs/03-dev-howtos/15-updating-test-database.md)
- [ ] Meets our accessibility [standards](https://github.com/guardian/recommendations/blob/e647ef695199ea3116ea20d827ef0f1364270a39/accessibility.md)
  - [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
  - [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#Keyboard)
  - [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#colour)

### CODE REVIEW

File: admin/app/AppLoader.scala [hunk 1]
```diff
@@ -39,15 +39,15 @@ trait AdminServices extends I18nComponents {
...
-  lazy val contentApiClient = wire[ContentApiClient]
-  lazy val ophanApi = wire[OphanApi]
-  lazy val emailService = wire[EmailService]
-  lazy val fastlyStatisticService = wire[FastlyStatisticService]
-  lazy val fastlyCloudwatchLoadJob = wire[FastlyCloudwatchLoadJob]
-  lazy val redirects = wire[RedirectService]
-  lazy val r2PagePressJob = wire[R2PagePressJob]
-  lazy val analyticsSanityCheckJob = wire[AnalyticsSanityCheckJob]
-  lazy val rebuildIndexJob = wire[RebuildIndexJob]
+  lazy val contentApiClient: ContentApiClient = wire[ContentApiClient]
```
Comment (mkurz) [inline admin/app/AppLoader.scala:42]: I think defining the type explicitly is necessary for `implicits` only.

**Review by mkurz — COMMENTED** [review]
LGTM

**Comment by ioannakok** [comment]
Closing this in favour of: https://github.com/guardian/frontend/pull/26806

File: admin/app/AppLoader.scala (hunk 1 above)
Comment (ioannakok) [reply admin/app/AppLoader.scala:42]: Yep, I closed this one in favour of: 
* https://github.com/guardian/frontend/pull/26806. 

Tried to use the [`scalafix ExplicitResultTypes`](https://scalacenter.github.io/scalafix/docs/rules/ExplicitResultTypes.html) here but didn't notice there was an `onlyImplicits` config.

**Pull request closed without merging**